tests/base_model_store_tests.py

```diff
@@ -121,6 +133,12 @@ class MyDerivedModel(MyBaseModel):
     union_date: Union[datetime, None] = field(default_factory=datetime.utcnow)
     bytes_type: Union[bytes, None] = None
     bytearray_type: Union[bytearray, None] = None
+    some_enum: SomeEnum = SomeEnum.FIELD_ONE
+    nested_enum: NestedModelWithEnum = field(
+        default_factory=lambda: NestedModelWithEnum(
+            some_str="string", some_enum=SomeEnum.FIELD_ONE
```

Comment:
To ensure that it's not unintentionally picking the other FIELD_ONE, can you next FIELD_TWO instead?